packages/state-management/src/reducer.ts

```diff
@@ -59,6 +59,12 @@ function siteinfoReducer(state = initialState.siteinfo, action: Action) {
                     codeInjectionBody:
                         decode(action.siteinfo.codeInjectionBody) ||
                         initialState.siteinfo.codeInjectionBody,
+                    mailingAddress:
```

Comment:
Why did we add `mailingAddress` here?

Reply:
Because `mailingAddress` is related to siteInfo.